app/server/src/core/plugins/mercurius.ts

```diff
@@ -90,7 +90,7 @@ export function attachMercuriusTo(server: FastifyInstance) {
         context: makeRequestContext,
         subscription: {
             context: makeSubscriptionContext,
-            emitter: process.env.NODE_ENV === 'production' ? redisEmitter : undefined,
+            emitter: redisEmitter,
```

Comment:
Leaving a comment to verify that this will now always run, even in development. Is this intended?

Reply:
It should now work with the emitter in development and testing. I'll make sure the students don't have issues running locally though just in case.